### PR TITLE
LPS-143393 | master

### PR DESCRIPTION
**Pull request by ozysouza**
https://issues.liferay.com/browse/LPS-143393

cc @david-gutierrez-mesa 
cc @Tim-Cao

### CODE REVIEW

File: portal-web/test/functional/com/liferay/portalweb/macros/ControlMenu.macro [hunk 1]
```diff
@@ -1,5 +1,11 @@
 definition {
 
+	macro toggleControls {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/macros/ControlMenu.macro:3]: Rename to switchToggleControls then move to WidgetPages.macro

File: portal-web/test/functional/com/liferay/portalweb/macros/ControlMenu.macro [hunk 2]
```diff
@@ -13,4 +19,10 @@ definition {
 			value1 = "${headerTitleLabel}");
 	}
 
+	macro viewToggleControls {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/macros/ControlMenu.macro:22]: move to WidgetPages.macro

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/widgetpage/ToggleControls.testcase [hunk 3]
```diff
@@ -41,15 +45,15 @@ definition {
 	test HideAndShowExistingInvisiblePortlet {
 		property portal.acceptance = "true";
 
-		task ("Can hide and show existing invisible portlets on widget page via switch toggle controls.") {
+		task ("View toggle controls is at view action") {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/widgetpage/ToggleControls.testcase:48]: View toggle controls is at view status

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/widgetpage/ToggleControls.testcase [hunk 4]
```diff
@@ -77,10 +79,10 @@ definition {
 			AssertNotVisible(
 				key_portletName = "Search",
 				locator1 = "ManagementBar#PORTLET_BODY");
+		}
 
-			Click(
-				key_status = "hidden",
-				locator1 = "ControlMenu#TOGGLE_CONTROLS");
+		task ("Switch Toggle Controls to unhidden then view portlets are shown") {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/widgetpage/ToggleControls.testcase:84]: task ("Switch Toggle Controls to view then invisible widgets are shown") {

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/widgetpage/ToggleControls.testcase [hunk 5]
```diff
@@ -41,15 +45,15 @@ definition {
...
-				siteName = "Test Site Name");
+				siteName = "Test Site Name 1");
 
-			AssertVisible(
-				key_status = "view",
-				locator1 = "ControlMenu#TOGGLE_CONTROLS");
+			ControlMenu.viewToggleControls(status = "view");
+		}
 
+		task ("Add a Web Content Display widget to page then view portlets are shown") {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/widgetpage/ToggleControls.testcase:56]: View invisible widgets are shown on page

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/widgetpage/ToggleControls.testcase [hunk 6]
```diff
@@ -61,14 +65,12 @@ definition {
 			AssertVisible(
 				key_portletName = "Search",
 				locator1 = "ManagementBar#PORTLET_BODY");
+		}
 
-			Click(
-				key_status = "view",
-				locator1 = "ControlMenu#TOGGLE_CONTROLS");
+		task ("Switch Toggle Controls to hidden then view portlets are not shown") {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/widgetpage/ToggleControls.testcase:70]: task ("Switch Toggle Controls to hidden then invisible widgets are not shown") {

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/widgetpage/ToggleControls.testcase [hunk 7]
```diff
@@ -111,15 +109,15 @@ definition {
 	test HideAndShowHeaderOfExistingVisiblePortlet {
 		property portal.acceptance = "true";
 
-		task ("Switch Toggle Controls to hidden") {
+		task ("View toggle controls is at view action") {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/widgetpage/ToggleControls.testcase:112]: task ("View toggle controls is at view status") {

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/widgetpage/ToggleControls.testcase [hunk 8]
```diff
@@ -226,32 +200,26 @@ definition {
 
 	@description = "This is a use case for LPS-108216. Switch toggle controls shouldn't affect portlets on content page."
 	@priority = "4"
-	test ToggleControlsNotAffectPortletsOnContentPage {
+	test NotAffectPortletsOnContentPage {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/widgetpage/ToggleControls.testcase:203]: ViewNotAffectWidgetsOnContentPage

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/widgetpage/ToggleControls.testcase [hunk 9]
```diff
@@ -287,8 +255,8 @@ definition {
 
 	@description = "This is a use case for LPS-108216. Assert Toggle Controls doesn't affect widget pages in other virtual instance scoped."
 	@priority = "4"
-	test ToggleControlsOnlyAffectCurrentInstance {
-		property test.name.skip.portal.instance = "ToggleControls#ToggleControlsOnlyAffectCurrentInstance";
+	test OnlyAffectCurrentInstance {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/widgetpage/ToggleControls.testcase:258]: ViewOnlyAffectCurrentInstance

**Review by Tim-Cao — COMMENTED** [review]
Hi @ozysouza, could you update according to comments? Thanks

**Comment by ozysouza** [comment]
> Hi @ozysouza, could you update according to comments? Thanks

Hi @Tim-Cao, PR updated following your comments. I removed the LocalizationWithSitePages because it was done in another LPS and I added here my mistake.

**Comment by Tim-Cao** [comment]
ci:test:sf

**Comment by Tim-Cao** [comment]
Hi @ozysouza, see changes https://github.com/liferay-echo/liferay-portal/compare/2cd29c284be25e8e512dbff815e7b98870ccef2c...0e17fa39aba0ea21504078a918d1c87f8944dcbf

**Comment by Tim-Cao** [comment]
ci:close

**Pull request closed without merging**